iphone/Maps/Bookmarks/Categories/Permissions/BMCPermissionsHeader.swift

```diff
@@ -28,8 +28,8 @@ final class BMCPermissionsHeader: UIView {
   var delegate: BMCPermissionsHeaderDelegate!
 
   private func updateButton() {
-    UIView.animate(withDuration: kDefaultAnimationDuration) {
-      self.button?.imageView?.transform = self.isCollapsed ? .init(rotationAngle: .pi) : .identity
+    UIView.animate(withDuration: 0.3) {
```

Comment:
Could you please leave kDefaultAnimationDuration as it was?

Reply:
If I keep kDefaultAnimationDuration button rotation animation will complete before tableview animation (animations are not synchronised).